frontend/src/components/viewed-cars/viewed-cars.tsx

```diff
@@ -0,0 +1,52 @@
+import { FC, useState } from 'react';
+
+import { ButtonFill } from '@components/common/button-fill/button-fill';
+import { ViewedCarsGrid } from '@components/viewed-cars/components/viewed-cars-grid/viewed-cars-grid';
+import { useGetHistoryOfViwedCarsQuery } from '@store/queries/history-viewed-cars';
+
+import styles from './styles.module.scss';
+
+import type { GetViewedCarsResponse } from '@autoline/shared';
+
+const ViewedCars: FC = () => {
+  const [takeCars, setTakeCars] = useState(6);
+
+  const params = {
+    userId: '0cdfe5ca-256f-49e4-855f-f438a4fac3c9',
+    skip: '0',
+    take: String(takeCars),
+  };
```

Comment:
This object will be recreated on each rerender and the `useGetHistoryOfViwedCarsQuery` will be restarted on each rerender as well